recipe/bld.bat

```diff
@@ -0,0 +1,20 @@
+mkdir build-shared
+if errorlevel 1 exit 1
+cd build-shared
+if errorlevel 1 exit 1
+
+:: Configure and install based on other `bld.bat`s and
+::https://github.com/google/crc32c/blob/master/.appveyor.yml
+cmake -G "Ninja" ^
+         -DBUILD_TESTING=OFF ^
+         -DBUILD_SHARED_LIBS=ON ^
+         -DCMAKE_BUILD_TYPE=Release ^
+         -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+         -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+         -DCRC32C_BUILD_TESTS=OFF ^
+         -DCRC32C_BUILD_BENCHMARKS=OFF ^
+         -DCRC32C_USE_GLOG=0 ^
```

Comment:
This might fix it.
```suggestion
         -DCRC32C_USE_GLOG=0 ^
         -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Reply:
That worked! https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=187510&view=logs&j=f97ca392-d626-52d0-4b2e-4b27aa319a77&t=7b750183-140d-50ae-3ee1-cbef6218bdf6

Thanks for the help.